modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -548,19 +564,43 @@ class FilePanel extends Component {
                   </table>
                 </div>
               </div>
-
             </div>
           </div>
         </Panel>
       </Panel>
     );
   }
+
+  /**
+   * Display a loading message when downloading files
+   */
+  showLoadingMessage() {
+      let type = 'success';
+      let title = 'We are currently working hard to download your files';
+      let text = 'Please be patient 😴';
```

Comment:
```suggestion
      let title = 'Your download will begin in a few moments';
      let text = 'Thank you for your patience';
```

Reply:
Hi @AlexandraLivadas  - Could you accept this change to the messaging ? thanks, 
Christine